experimental/sample_web_static/build_static_emscripten_demo.sh

```diff
@@ -76,7 +76,7 @@ emcmake "${CMAKE_BIN?}" -G Ninja .. \
   -DIREE_BUILD_COMPILER=OFF \
   -DIREE_BUILD_TESTS=OFF
 
-"${CMAKE_BIN?}" --build . --target \
+"${CMAKE_BIN?}" --build . --target \ -- -k 0
```

Comment:
Whoops, sorry! I forgot about continuations when constructing my regex and was relying on CI to catch things. In fact, this can't be before the continuation as it needs to be at the end of the command after all arguments that CMake parses itself. Let's just revert this file

Reply:
"Relying" is a strong word. I did scan the diff and missed this

Reply:
I can also include a revert in my next PR touching this code. It's in experimental/ for a reason :D